ui/grpc-web/etc/localhost.crt

```diff
@@ -0,0 +1,18 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
checking in certs and keys is a bad practice. It is compromised now. Not sure how it affects you but I my knee-jerk reaction is revoking the cert and adding it to .gitignore.

Reply:
this is publicly accessible certificate file from grpc-web project. i plan to remove grpc-web folder in the future.